java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/spark/SparkJobManager.scala

```diff
@@ -36,12 +36,21 @@ object SparkJobManager {
    * @return the SparkLauncher object
    */
   def createSparkLauncher(mainClass: String): SparkLauncher = {
-
-    val launcher = new SparkLauncher()
+    val env: java.util.Map[String, String] = new java.util.HashMap[String, String]
+    // config may empty, need check
+    if (!TaskManagerConfig.isEmpty(TaskManagerConfig.HADOOP_CONF_DIR)) {
+      env.put("HADOOP_CONF_DIR", TaskManagerConfig.HADOOP_CONF_DIR)
```

Comment:
test taskmanager conf file > system env

Reply:
Test on new taskmanager, check by classpath print of zk in log file: if conf file, HADOOP_CONF_DIR in conf file is in the print, if no conf file, use the system env.
But the system env is passed by taskmanager too. So I test one more case:
- start taskmanager
- set env
- submit spark job, use the env set by taskmanager config or the system env in step 2.